### ISSUE TITLE
Subject metadata is invisible in VoiceOver

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#649]
**Package**
lib-classifier

**Feature or Issue Description**
Subject metadata is announced as a table with one column and zero rows in VoiceOver and can't be entered or read out.
![Screenshot showing the Subject Metadata table in VoiceOver.](https://user-images.githubusercontent.com/59547/55548254-f973a900-56ca-11e9-80be-40ed8f781111.png)

**Comment by rogerhutchings** [comment]
This will be an issue with the underlying Grommet `DataTable` component; we can write our own, submit a PR to grommet, or find an alternative lib.

**Comment by eatyourgreens** [comment]
Grommet's storybook DataTable works just fine in VoiceOver. I can enter it and navigate the data.
You can try it here: https://storybook.grommet.io/?path=/story/datatable--tunable-datatable
![Screenshot showing a data table announce correctly in VoiceOver.](https://user-images.githubusercontent.com/59547/55549093-fda0c600-56cc-11e9-875e-d733821e346a.png)

**Comment by rogerhutchings** [comment]
Well that's weird, [the modal](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/MetaTools/components/MetadataModal/MetadataModal.js) looks like it's using a pretty vanilla DataTable

**Comment by srallen** [comment]
There may be props that are missing that need to be set or it's possible we need to bump Grommet again as now they're on 2.6... I'll check the changelog.

**Comment by eatyourgreens** [comment]
I took a quick look at the generated DOM and couldn't see anything obviously wrong with the table.

**Comment by srallen** [comment]
Maybe the problem is that it is in the modal?

**Comment by eatyourgreens** [comment]
Could be. I thought maybe VoiceOver is reading the table before it's fully rendered in the DOM.

**Comment by eatyourgreens** [comment]
Firefox dev tools expose the accessibility tree for the document so I had a look at this table and its children, but couldn't see anything obviously wrong. All the rows were listed as expected.
